docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -6,7 +6,7 @@ metadata:
     CourtFormsOnline
   title url: https://courtformsonline.org
   logo: |
-    <img src="/packagestatic/docassemble.MAVirtualCourt/mass_icon.png" class="ma_icon" alt="MA State Icon"><div class="title">CourtFormsOnline: MassAccess</div>
+    <img src="/packagestatic/docassemble.AssemblyLine/ma_logo.png" class="ma_icon" alt="MA State Icon"><div class="title">CourtFormsOnline: MassAccess</div>
```

Comment:
Suggestion: can we change the alt text (and maybe the filename, but less important) so it matches? Maybe just "Assembly Line Logo" as the new text?

Let's also remove the text "MassAccess", and just keep "CourtFormsOnline"

```suggestion
    <img src="/packagestatic/docassemble.AssemblyLine/ma_logo.png" class="ma_icon" alt="Assembly Line Logo"><div class="title">CourtFormsOnline</div>
```